### PR TITLE
Minor fix to registry

### PR DESCRIPTION
**Pull request by BorisKozo**
Fixing issue #77

Added some unit tests for the type registry
Fixed 2 typos in existing unit tests that were found due to this change

### CODE REVIEW

**Comment by BorisKozo** [comment]
Moved from #83

File: src/backbone.syphon.typeregistry.js [hunk 1]
```diff
@@ -33,7 +37,7 @@ _.extend(TypeRegistry.prototype, {
 
   // Remove an item from a given type registration
   unregister: function(type) {
-    if (this.registeredTypes[type]) {
+    if (_.has(this.registeredTypes,type)) {
```
Comment (rhubarbselleven) [inline src/backbone.syphon.typeregistry.js]: Please pop in a space for formatting:

`if (_.has(this.registeredTypes, type)) {`

**Comment by rhubarbselleven** [comment]
Great stuff @BorisKozo. Could you please squash this down to 1 commit?

File: src/backbone.syphon.typeregistry.js (hunk 1 above)
Comment (BorisKozo) [inline src/backbone.syphon.typeregistry.js]: oops, forgot to reformat document

Comment (rhubarbselleven) [inline src/backbone.syphon.typeregistry.js]: no probs, once #88 lands, this kind of stuff will be picked up on `grunt test`

**Comment by rhubarbselleven** [comment]
Special Sauce. Thanks @BorisKozo

**Pull request merged**